src/actions/app.js

```diff
@@ -32,6 +32,18 @@ export function profilePublished(hash: string): Action {
   };
 }
 
+export function setProfileSharingStatus(
+  profileSharingStatus: ProfileSharingStatus = {
+    sharedWithUrls: false,
+    sharedWithoutUrls: false,
+  }
```

Comment:
I don't think you need the default values here.
If you looked at Redux doc, maybe you confused with the default values for the _reducer_, but here it's the _action creator_, different beast :-)